Charm/interpreter/interpreter.py

```diff
@@ -1066,16 +1062,51 @@ def solveDetermined(self):
         #     pickle.dump(results, ofile)
         #     ofile.close()
         self.result = results
-        self.vars=flat_iter_vars
-        logging.log(logging.INFO, 'Results saved to {}'.format(file_name))
-        logging.log(logging.INFO, 'Time used: {}'.format(end - start))
+        self.variables = iter_vars
+        self.values = iter_vals
+        self.flat_variables=flat_iter_vars
+        # logging.log(logging.INFO, 'Results saved to {}'.format(file_name))
+        # logging.log(logging.INFO, 'Time used: {}'.format(end - start))
 
     def __plot(self, node: PlotNode):
         if node.dependent not in self.targets:
             logging.error("Var {} not in explored targets, cannot be plotted".format(node.dependent))
             return
-        dependent_variable_index=self.targets.index(node.dependent)
-        x,y=[],[]
+        dependent_variable_index = self.targets.index(node.dependent)
```

Comment:
Do you mind adding some comments below to explain the logics? Might help others in the future :)